operator/src/main/resources/scripts/stop-server.py

```diff
@@ -36,29 +43,42 @@ def getEnvVar(var):
 file.write(decoded)
 file.close()
 
-# Connect to nodemanager and stop server
+def shutdownUsingNodeManager(domainName, domainDir):
+  try:
+    nmConnect(userConfigFile='/weblogic-operator/introspector/userConfigNodeManager.secure',
+              host='127.0.0.1',port='5556',
+              domainName=domainName,
+              domainDir=domainDir,
+              nmType='plain')
+  except Exception, e:
+    print e
+    print('Failed to connect to the node manager')
+    exit(exitcode=2)
+
+# Connect to the server and request that it shuts down
 
 try:
-  nmConnect(userConfigFile='/weblogic-operator/introspector/userConfigNodeManager.secure',
+  connect(userConfigFile='/weblogic-operator/introspector/userConfigNodeManager.secure',
             userKeyFile='/tmp/userKeyNodeManager.secure.bin',
-            host='127.0.0.1',port='5556',
+            url=connect_url,
             domainName=domain_name,
             domainDir=domain_path,
             nmType='plain')
```

Comment:
Ideally, the on-line call could be done via REST instead of a shutdown script.  This would eliminate the need to wait for WLST to spin up during shutdown.

Reply:
@tbarnes-us, I'd like to punt on this now if you will agree.  I have some code that makes the REST call and it mostly works, but I'm seeing some hard to diagnose HTTP 500 errors and then I just don't have the code (right) yet to watch for the server to finish shutting down before deleting the pod.  So, I'd like to merge this change and then work on that as a separate PR.

Reply:
Sounds good to me.  Would be nice to track the separate PR via a JIRA if you aren't going to get to it soon.